README.md

```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
 
-```bash
-furyctl version
-INFO[0000] Furyctl version 0.8.0
-INFO[0000] built 2022-09-13T16:52:19Z from commit b3c9df89803ed7a4fe4a05d6c3eb859cc00276b6
+```console
+$ furyctl version
+buildTime: 2023-01-13T09:50:15Z
+gitCommit: 349c14a06dd6163b308e4e8baa47ec9cc59712e1
+goVersion: go1.19
+osArch: amd64
+version: 0.9.0
 ```
 
+## Usage
+
+See all the available commands and their usage by running `furyctl help`.
+
 > 💡 **TIP**
 >
-> Enable autocompletion for `furyctl` cli on your shell (currently autocompletion is supported for `bash`, `zsh`, `fish`).
-> To see the instruction to enable it, run `furyctl completion -h`
+> Enable command tab autocompletion for `furyctl` on your shell (`bash`, `zsh`, `fish` are supported).
+> See the instruction on how to enable it with `furyctl completion --help`
 
-## Usage
+<!-- line left blank as spacer -->
 
-See the available commands with `furyctl --help`:
+> **Warning**
+> (furyctl-ng alpha version only)
+>
+> `furyctl-ng` is compatible with KFD versions 1.22.1, 1.23.3 and 1.24.0, but you will need to use the flag `--distro-location 'git::git@github.com:sighupio/fury-distribution.git?depth=1&ref=feature/furyctl-next'`
+> in *every command* until the next release of the KFD.
 
-```bash
-furyctl --help
+### Basic Usage
+
+Basic usage of `furyctl` for a new project consists on the following steps:
+
+1. Creating a configuration file defining the prequired infrastructure, Kubernetes cluster details, and KFD modules configuration.
+2. Creating a cluster as defined in the configuration file.
+3. Destroying the cluster and its related resources.
+
+#### 1. Create a configuration file
+
+`furyctl` provides a command that outputs a sample configuration file (by default called `furyctl.yaml`) with all the possible fields explained in comments.
+
+furyctl configuration files have a kind, that specifies what type of cluster will be created, for example the `EKSCluster` kind has all the parameters needed to create a KFD cluster using the EKS managed clusters from AWS.
 
-A command-line tool to manage cluster deployment with Kubernetes
+Additionaly, the schema of the file is versioned with the `apiVersion` field, so when new features are introduced you can switch to a newer version of the configuration file structure.
 
-Usage:
-  furyctl [command]
+To create a sample configuration file as a starting point use the following command:
 
-Available Commands:
-  bootstrap   Creates the required infrastructure to deploy a battle-tested Kubernetes cluster, mostly network components
-  cluster     Creates a battle-tested Kubernetes cluster
-  completion  Generate completion script
-  help        Help about any command
-  init        Initialize the minimum distribution configuration
-  vendor      Download dependencies specified in Furyfile.yml
-  version     Prints the client version information
+```console
+furyctl create config --version <KFD version> --kind "EKSCluster"
```

Comment:
If `--distro-ocation` is mandatory, i would add that in all the example commands, for now

Reply:
Added the flag to all commands, it looks a little ugly but let's hope that we can remove it sooner than later.